instrumentation/active_job/lib/opentelemetry/instrumentation/active_job/handlers/enqueue.rb

```diff
@@ -28,10 +19,22 @@ def initialize(...)
           # @return [Hash] with the span and generated context tokens
           def start_span(name, _id, payload)
             job = payload.fetch(:job)
-            span = tracer.start_span(@span_name_formatter.call(job), kind: :producer, attributes: @mapper.call(payload))
+            span = tracer.start_span(span_name(job), kind: :producer, attributes: @mapper.call(payload))
             OpenTelemetry.propagation.inject(job.__otel_headers) # This must be transmitted over the wire
             { span: span, ctx_token: OpenTelemetry::Context.attach(OpenTelemetry::Trace.context_with_span(span)) }
           end
+
+          private
+
+          def span_name(job)
```

Comment:
I think it'd be best to move this to the `Default` class and also change all the other spans to have a consistent naming, but I'm not sure if we want to lump this change in the PR, this was merely intended to fix a regression